packages/strapi-generate-new/json/package.json.js

```diff
@@ -7,80 +7,86 @@
 // Public node modules.
 const _ = require('lodash');
 
-const { packageManager } = require('strapi-utils');
-
 /**
  * Expose main package JSON of the application
  * with basic info, dependencies, etc.
  */
 
 module.exports = scope => {
   const cliPkg = scope.strapiPackageJSON || {};
-  // Store the package manager info into the package.json
-  const pkgManager = packageManager.isStrapiInstalledWithNPM() ? 'npm' : 'yarn';
 
   // Let us install additional dependencies on a specific version.
   // Ex: it allows us to install the right version of knex.
-  const additionalsDependencies = _.isArray(scope.additionalsDependencies) ?
-    scope.additionalsDependencies.reduce((acc, current) => {
-      const pkg = current.split('@');
-      const name = pkg[0];
-      const version = pkg[1] || 'latest';
+  const additionalsDependencies = _.isArray(scope.additionalsDependencies)
+    ? scope.additionalsDependencies.reduce((acc, current) => {
+        const pkg = current.split('@');
+        const name = pkg[0];
+        const version = pkg[1] || 'latest';
 
-      acc[name] = name.indexOf('strapi') !== -1 ? getDependencyVersion(cliPkg, 'strapi') : version;
+        acc[name] =
+          name.indexOf('strapi') !== -1
+            ? getDependencyVersion(cliPkg, 'strapi')
+            : version;
 
-      return acc;
-    }, {}) : {};
+        return acc;
+      }, {})
+    : {};
 
   // Finally, return the JSON.
   return _.merge(scope.appPackageJSON || {}, {
-    'name': scope.name,
-    'private': true,
-    'version': '0.1.0',
-    'description': 'A Strapi application.',
-    'main': './server.js',
-    'scripts': {
-      'dev': 'strapi dev',
-      'start': 'strapi start',
-      'build': 'strapi build',
-      'strapi': 'strapi', // Allow to use `npm run strapi` CLI,
-      'lint': 'eslint api/**/*.js config/**/*.js plugins/**/*.js'
+    name: scope.name,
+    private: true,
+    version: '0.1.0',
+    description: 'A Strapi application.',
+    main: './server.js',
```

Comment:
I think we should remove this line since this file doesn't exist now.

Reply:
Yes sure